sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.validateOffering.e2e-spec.ts

```diff
@@ -231,6 +201,543 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-validateOffering",
+    const payload = {
+      offeringName: "Offering validation",
+      yearOfStudy: 1,
+      offeringIntensity: OfferingIntensity.fullTime,
+      offeringDelivered: OfferingDeliveryOptions.Online,
+      hasOfferingWILComponent: "no",
+      studyStartDate: "2024-05-23",
+      studyEndDate: "2024-08-16",
+      lacksStudyBreaks: true,
+      studyBreaks: [],
+      offeringType: OfferingTypes.Public,
+      offeringDeclaration: true,
+      actualTuitionCosts: 1234,
+      programRelatedCosts: 3211,
+      mandatoryFees: 456,
+      exceptionalExpenses: 555,
+    };
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        offeringStatus: "Approved",
+        errors: [],
+        infos: [],
+        warnings: [],
+        studyPeriodBreakdown: {
+          fundedStudyPeriodDays: 86,
+          totalDays: 86,
+          totalFundedWeeks: 13,
+          unfundedStudyPeriodDays: 0,
+        },
+      });
+  });
+
+  it("Should validate an offering when valid data with study breaks is passed in for a full-time education program.", async () => {
+    // Arrange
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+
+    const endpoint = `/institutions/education-program-offering/location/${collegeFLocation.id}/education-program/${collegeFFullTimeProgram.id}/validation`;
+    const payload = {
+      offeringName: "Offering validation",
+      yearOfStudy: 1,
+      offeringIntensity: OfferingIntensity.fullTime,
+      offeringDelivered: OfferingDeliveryOptions.Online,
+      hasOfferingWILComponent: "no",
+      studyStartDate: "2024-01-01",
+      studyEndDate: "2024-08-16",
+      lacksStudyBreaks: false,
+      studyBreaks: [
+        {
+          breakStartDate: "2024-08-05",
+          breakEndDate: "2024-08-16",
+        },
+      ],
+      offeringType: OfferingTypes.Public,
+      offeringDeclaration: true,
+      actualTuitionCosts: 1234,
+      programRelatedCosts: 3211,
+      mandatoryFees: 456,
+      exceptionalExpenses: 555,
+    };
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        offeringStatus: "Approved",
+        errors: [],
+        infos: [],
+        warnings: [],
+        studyPeriodBreakdown: {
+          fundedStudyPeriodDays: 229,
+          totalDays: 229,
+          totalFundedWeeks: 33,
+          unfundedStudyPeriodDays: 0,
+        },
+      });
+  });
+
+  it("Should validate an offering with info when valid data with study breaks is passed in for a full-time education program.", async () => {
```

Comment:
In description for all E2E tests which are part of this PR, please update `full-time education program` to `full-time education program offering` and like wise for offering.